Source/App.js

```diff
@@ -1 +1,340 @@
+                    showBackground : true,
+                    scale : 0.6,
+                    horizontalOrigin : Cesium.HorizontalOrigin.CENTER,
+                    verticalOrigin : Cesium.VerticalOrigin.BOTTOM,
+                    distanceDisplayCondition : new Cesium.DistanceDisplayCondition(10.0, 8000.0),
+                    disableDepthTestDistance : Number.POSITIVE_INFINITY
+                };
+            }
+        }
+    });
+
+    // Load a drone flight path from a CZML file
+    var dronePromise = Cesium.CzmlDataSource.load('./Source/SampleData/SampleFlight.czml');
+
+    // Save a new drone model entity
+    var drone;
+    dronePromise.then(function(dataSource) {
+        viewer.dataSources.add(dataSource);
+        drone = dataSource.entities.values[0];
+        // Attach a 3D model
+        drone.model = {
+            uri : './Source/SampleData/Models/CesiumDrone.gltf',
+            minimumPixelSize : 128,
+            maximumScale : 2000
+        };
+        // Add computed orientation based on sampled positions
+        drone.orientation = new Cesium.VelocityOrientationProperty(drone.position);
+
+        // Smooth path interpolation
+        drone.position.setInterpolationOptions({
+            interpolationAlgorithm : Cesium.HermitePolynomialApproximation,
+            interpolationDegree : 2
+        });
+        drone.viewFrom = new Cesium.Cartesian3(30, 0, 0);
+    });
+
+    //////////////////////////////////////////////////////////////////////////
+    // Load 3D Tileset
+    //////////////////////////////////////////////////////////////////////////
+
+    // Load the NYC buildings tileset
+    var city = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
+        url: 'https://cesiumjs.org/NewYork/3DTilesGml',
+        maximumScreenSpaceError: 16 // default value
+    }));
+
+    // Adjust the tileset height so it's not floating above terrain
+    var heightOffset = -32;
+    city.readyPromise.then(function(tileset) {
+        // Position tileset
+        var boundingSphere = tileset.boundingSphere;
+        var cartographic = Cesium.Cartographic.fromCartesian(boundingSphere.center);
+        var surfacePosition = Cesium.Cartesian3.fromRadians(cartographic.longitude, cartographic.latitude, 0.0);
+        var offsetPosition = Cesium.Cartesian3.fromRadians(cartographic.longitude, cartographic.latitude, heightOffset);
+        var translation = Cesium.Cartesian3.subtract(offsetPosition, surfacePosition, new Cesium.Cartesian3());
+        tileset.modelMatrix = Cesium.Matrix4.fromTranslation(translation);
+    });
+
+    //////////////////////////////////////////////////////////////////////////
+    // Style 3D Tileset
+    //////////////////////////////////////////////////////////////////////////
+
+    var defaultStyle = new Cesium.Cesium3DTileStyle({
+        color : "color('white')",
+        show : true
+    });
+
+    city.readyPromise.then(function(tileset) {
+        tileset.style = defaultStyle;
+    });
```

Comment:
I think you can set the style before the tileset is ready, but double check that.